src/app/notes/notes-list/notes-list.component.html

```diff
@@ -1,33 +1,27 @@
 <div *ngIf="!loading">
-  <div *ngIf="notes?.length > 0">
-    <div class="sort">
-      <button
-        mat-icon-button
-        (click)="reverse()"
-        matTooltip="Show Oldest Notes First"
-      >
+    <div *ngIf="notes?.length > 0">
+        <div class="sort">
+            <form class="example-form">
+                <mat-form-field class="example-full-width">
+                    <mat-label>Search Notes by Title</mat-label>
+                    <input matInput name="search" [(ngModel)]="searchText" autocomplete="off" />
+                </mat-form-field>
+            </form>
+            <button mat-icon-button (click)="reverse()" matTooltip="Show Oldest Notes First">
```

Comment:
```suggestion
            <button mat-icon-button (click)="reverse()" [matTooltip]="sortTooltipText">
```
We should change the tooltip text based on the current sort direction. If a user clicks the sort icon button once, 'oldest first' is not really correct anymore 🚀